jack-matchmaker.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=auto-connect JACK ports matching given patterns
+
+[Service]
+EnvironmentFile=/etc/conf.d/jack-matchmaker
+ExecStart=/usr/bin/jack-matchmaker -p $PATTERN_FILE -e $EXACT_MATCHING -N $CLIENT_NAME -m MAX_ATTEMPTS -v $VERBOSITY
```

Comment:
What syntax is supported here? Does this support Bash variable expansion syntax e.g.

     ${PATTERN_FILE+-p $PATTERN_FILE}

so you only pass command line options that are defined in the environment file?

Reply:
Also, do these placeholders need quoting for values with whitespace in them?

Reply:
Parameter expansion works by passing the commands to `/bin/sh -c` and escaping the `$` with `$$` so systemd doesn't do its filtering.

Yes they do need quoting.